src/views/PageSearchPortal.vue

```diff
@@ -90,31 +95,46 @@ export default {
         },
     },
     async created() {
-        const currentTab =
-            this.activeTab === 0 ? 'parking-request' : 'interested-request';
-        this.$router.push({
-            path: this.$route.fullPath,
-            query: { tab: currentTab },
-        });
-        this.getAgents();
-        const mobile = this.$route.query['mobile'];
-        if (mobile) {
-            this.updateMobileInput(mobile);
+        const tab = this.$route.query['tab'];
+        if (tab) {
+            if (tab === 'interested-request') {
+                this.updateActiveTab(1);
+                if (this.$route.query['latlng']) {
+                    const latlng = this.$route.query['latlng'];
+                    this.updateSOLatLngInput(latlng);
+                    this.getInterestedVO(this.SOLatLngInput);
+                } else {
+                    this.getParkingRequests();
+                }
+            } else {
+                this.updateActiveTab(0);
+                const mobile = this.$route.query['mobile'];
+                if (mobile) {
+                    this.updateMobileInput(mobile);
+                    this.getParkingRequests(mobile);
```

Comment:
now this method is in store
and we are already setting the mobile state in store so do we need to pass this as arg.

Reply:
@arrogant-ant I've completed all the changes. Could you review them once?